### ISSUE TITLE
fix: Radio button and platform radio group accessibility fix

### ISSUE THREAD

**Issue by DeepakSap14** [SAP/fundamental-ngx#2999]
#### Is this a bug, enhancement, or feature request?
enhancement

#### Briefly describe your proposal.
Currently radio button does not have aria-* tags, tab-order and keyboard accessibility.

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
Angular 10

#### Is there anything else we should know?
accessibility requirements: 
https://app.gitbook.com/@fundamental-ngx/s/docs/accessibility-requirements/radio-button

**Comment by DeepakSap14** [comment]
PR https://app.zenhub.com/workspace/o/sap/fundamental-ngx/issues/3036 has been fixed and merged. it fixes this issue. hence closing.
